Adjusted syntax according to scalastyle-config

abejgonzalez opened the pull request:
Updated most of the syntax to match the scalastyle config that is currently present. This is part of the work to get Travis CI up and running with no errors (this will be one of the things that is run during Travis).

abejgonzalez commented:
A quick thought: Do we want to have a different scalastyle config (shorter line width, etc)? Maybe that can be a question for later? This is just to get us started.

abejgonzalez commented:
@ccelio Im assuming we can change the headers to say 2018 instead of 2015. Is that fine?

ccelio commented:
120 line width comes from Java. I think it's a fine size for two-wide on modern monitors.

I can't speak to what to do with copyright headers; I guess update them?

I wouldn't blindly sed update 2015 to 2018 though. Looks like you hit some other uses.

abejgonzalez commented:
What do you mean by "looks like you hit some other uses". As for the headers, I thought it would be fine to update whenever the file has changed (so for the Chisel3 update most files were changed hence the 2015->2018 change).

sagark commented:
AFAIK these headers should be [first written year - now], you’re not
supposed to just change the year to the current year.

On Wed, Dec 12, 2018 at 12:19 AM Abraham Gonzalez <notifications@github.com>
wrote:

> What do you mean by "looks like you hit some other uses". As for the
> headers, I thought it would be fine to update whenever the file has changed
> (so for the Chisel3 update most files were changed hence the 2015->2018
> change).
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/riscv-boom/riscv-boom/pull/136#issuecomment-446500778>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA4eQq4E-mIsmt2fGordjwM0LvII_zm2ks5u4LwFgaJpZM4ZOySt>
> .
>

ccelio commented on src/main/scala/bpu/brpredictor.scala:
```diff
@@ -1,14 +1,15 @@
 //------------------------------------------------------------------------------
 //
 // Christopher Celio
-// 2015 Oct 12
```
Sed script collateral damage.

ccelio commented on src/main/scala/bpu/gshare.scala:
```diff
@@ -1,14 +1,15 @@
 //------------------------------------------------------------------------------
 //
 // Christopher Celio
-// 2015 Apr 28
```
Another victim. F.

abejgonzalez commented:
Ahh, ok I will change the scripts back to have a range and mention authors

abejgonzalez commented:
Ok, I moved the authors to the top headers and fixed the dates so that they are consistent from before (if the file did not have a date I just specified it as 2015)

ccelio commented:
Looks good to me.
